### ISSUE TITLE
Multiple Instances of the applicaiton opened on the same platform is not updated

### ISSUE THREAD

**Issue by dominic2412** [dominic2412/pe#7]
**ISSUE**

In EventfulNUS, when multiple instances of the application are open, changes made in one window do not reflect in the other. This can lead to data inconsistency, particularly for users who manage a high volume of event data across multiple faculties, roles, or participants[which is mentioned in the UG and DG]. A user might open multiple instances of EventfulNUS, intending to work on different aspects of the event data in parallel (e.g., managing participants in one window while scheduling events in another). However, any changes made in one instance, such as adding a new participant or updating event details, remain isolated to that specific window. 
The manager for this should be very busy also, most people working/ study have multiple tabs like 100 tabs open, not to mention about people working to handle the IFG, a cross-faculty University level game
This could lead the user to mistakenly assume that their updates were applied globally across all instances.

Further, there was mentioning in the UG that the data is saved automatically, which could be confusing to users.

![image.png](https://raw.githubusercontent.com/dominic2412/pe/main/files/b68c2505-51bc-4342-a3af-962219dc365b.png)

**Demostration**
<i><video controls><source src="https://raw.githubusercontent.com/dominic2412/pe/main/files/ae9079c3-7d51-4fac-a962-7bd253a2e010.mp4" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/dominic2412/pe/main/files/ae9079c3-7d51-4fac-a962-7bd253a2e010.mp4</i>

**Reason for severity**

This issue impacts the reliability of the application, particularly in event management scenarios where accuracy and synchronization are essential, such as coordinating participants, updating event details, or assigning roles. For busy event organisers, managing a large volume of data across faculties or roles, the risk of data discrepancies becomes particularly significant.

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Hello, thank you for the bug report! While we know that users may want to use multiple instances of the app and edit the lists in parallel, we see this feature as **"Out Of Scope"** for the current version of our project. For now, we believe it's suitable for our users to stick to one application instance at a time.

In addition, we also believe that this violates some of the project constraints. For example, we are not allowed to use a DBMS (Constraint-No-DBMS) (Database Management System like MySQL) to store data. However, to have data changes flow consistently among multiple instances of the app, one would require a DBMS. 
The need of a backend also violates the Constraint-No-Remote-Server.

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.High`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
